open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -517,6 +519,7 @@ SETUP
  TRUEFALSECHR(FALSE)="false"
  TRUEFALSECHR(TRUE)="true"
  SACSHAREGROUPSMAX=19
+ CONFIGPROGRAMMINDATE='10/20/22'
```

Comment:
Maybe group this version information with the other version variables below.  And maybe have all the version information as the first thing in the setup division.

Group CONFIGPROGRAMMINDATE with these: 
`PROGRAMVERSION="1.1.0"
 LASTMODDATE='10/20/22'
 LASTMODTIME="12:00 MT"
 PROGRAMUPDATENOTE1="Copy name functionality added"
 PROGRAMUPDATENOTE2="Error code output update, JSON code fix"`

Reply:
Done